packages/docs-app/src/examples/core-examples/buttonGroupExample.tsx

```diff
@@ -8,37 +8,46 @@ import * as React from "react";
 
 import { AnchorButton, Button, ButtonGroup, Switch } from "@blueprintjs/core";
 import { BaseExample, handleBooleanChange } from "@blueprintjs/docs-theme";
+import { AlignmentSelect } from "./common/alignmentSelect";
 
 export interface IButtonGroupExampleState {
-    fill?: boolean;
-    minimal?: boolean;
-    large?: boolean;
-    vertical?: boolean;
+    align: "left" | "center" | "right";
+    fill: boolean;
+    iconOnly: boolean;
+    minimal: boolean;
+    large: boolean;
+    vertical: boolean;
 }
 
 export class ButtonGroupExample extends BaseExample<IButtonGroupExampleState> {
     public state: IButtonGroupExampleState = {
+        align: "center",
         fill: false,
+        iconOnly: false,
         large: false,
         minimal: false,
```

Comment:
small in this example would be neat

Reply:
ButtonGroup doesn't have a `small` prop. 
let's wait on that, i want to add modifier props _everywhere_